markdown/events/2022/hackathon-october-2022.md

```diff
@@ -17,4 +17,54 @@ This hackathon will be held in advance of the _**Nextflow Summit 2022**_ in Barc
 
 In addition to the in-person event, we will endeavour to run a hybrid option online in [gather.town](https://gather.town/).
 
-Please check back here soon - more details will be added to this page soon as we continue to plan the event.
+This hackathon is not meant to be a training event but aimed at people that already have a basic understanding of nextflow and/or nf-core. For anyone interested in learning nextflow, there are going to be workshops in the week before the hackathon. As soon as we have more information, we will let you know.
```

Comment:
Are they happening? The summit is after the hackathon iirc

Reply:
> Are they happening?
That was what I understood from our discussion last Friday at least. Maybe we can wait until the next meeting to ask again?

> The summit is after the hackathon iirc
Hackathon first, then summit, right? That's what I say, I believe?

Reply:
yes ok that is why I was wondering about the workshop before the hackathon. Or is that yet anothe event?

Reply:
no, it's the workshops we talked about and that are mentioned in the meeting notes.

Reply:
Suggestion updated 👍🏻

Reply:
x-ref: https://github.com/nf-core/nf-co.re/pull/1203